packages/core/test/Posting.ts

```diff
@@ -269,15 +269,16 @@ describe('Post', function () {
                 const hashedContent = ethers.utils.keccak256(
                     ethers.utils.toUtf8Bytes(content)
                 )
-                const receipt = await unirepSocialContract
+                const { logs } = await unirepSocialContract
```

Comment:
You can make an assertion about the events emitted by the contract using hardhat/chai. It's much cleaner to use [this](https://github.com/Unirep/Unirep/blob/v1.1/packages/contracts/test/epochTransition.test.ts#L565) kind of syntax i think (also supports data params)

Reply:
I think here we only need `postId` but we don't have to make assertion.
So I parse the logs here to get `postId`

Reply:
Ah sure, that makes sense.